2018 Robot/src/org/usfirst/frc/team4999/robot/RobotMap.java

```diff
@@ -52,11 +55,14 @@
 	
 	public static NetworkTable pidTable = NetworkTableInstance.getDefault().getTable("PID");
 	
-	public static PowerDistributionPanel pdp = new PowerDistributionPanel();
+	//TODO: Debug pdp
+	//public static PowerDistributionPanel pdp = new PowerDistributionPanel();
```

Comment:
Why commented out?  Isn't this needed to talk to the lights on the PDP?

Reply:
It was giving CAN bus errors and crashing robot. It needs to be investigated